grpc/send_incoming_call_notification.go

```diff
@@ -0,0 +1,69 @@
+package grpc
+
+import (
+	"bytes"
+	"context"
+	"encoding/binary"
+	"log"
```

Comment:
This is only relevant once your plogger PR is merged, and I'll only write this once despite it being in multiple files

For official logging we prefer to stick with plogger whenever possible, that way all servers log files are consistent and have all information required to look into issues